test/unittests/instructions_test.cpp

```diff
@@ -327,3 +327,17 @@ TEST(instructions, berlin_hard_fork)
     EXPECT_EQ(b[OP_STATICCALL].gas_cost, 100);
     EXPECT_EQ(b[OP_SLOAD].gas_cost, 100);
 }
+
+TEST(instructions, london_hard_fork)
+{
+    const auto l = evmc_get_instruction_metrics_table(EVMC_LONDON);
+    const auto b = evmc_get_instruction_metrics_table(EVMC_BERLIN);
+    const auto ln = evmc_get_instruction_names_table(EVMC_LONDON);
+    const auto bn = evmc_get_instruction_names_table(EVMC_BERLIN);
+
+    for (int op{OP_STOP}; op <= OP_SELFDESTRUCT; ++op)
```

Comment:
I know this isn't introduced by this PR as a new thing, but why is this going from STOP->SELFDESTRUCT and not `0` to `255` or some `min`/`max` based on the enum type? I know it is identical but still feels weird basing it on the instruction name.

Reply:
Make sense, I will change that later.